test/uk/gov/hmrc/claimvatenrolmentfrontend/services/JourneyValidateServiceSpec.scala

```diff
@@ -27,45 +27,78 @@ import uk.gov.hmrc.claimvatenrolmentfrontend.config.AppConfig
 import scala.concurrent.{ExecutionContextExecutor, Future}
 
 class JourneyValidateServiceSpec extends AnyWordSpec with GuiceOneAppPerSuite with Matchers with MockJourneyDataRepository
-                with MockJourneySubmissionRepository with FeatureSwitching {
+                with MockJourneySubmissionRepository with FeatureSwitching with PrivateMethodTester with MockJourneyValidationService {
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
   implicit val request: Request[_] = FakeRequest()
   implicit val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
 
   implicit val ec: ExecutionContextExecutor = scala.concurrent.ExecutionContext.global
 
-  class TestService(flag: Boolean) extends JourneyValidateService(mockJourneyDataRepository, mockJourneySubmissionRepository, appConfig){
-    override def isJourneyLocked(journeyId: String, authInternalId: String)(implicit request: Request[_]) =
-      Future.successful(flag)
-  }
+  object TestService extends JourneyValidateService(mockJourneyDataRepository, mockJourneySubmissionRepository, appConfig)
 
-  "the journey continue" when {
-    "knownFactsCheck is disabled" in {
-      disable(KnownFactsCheckFlag)
+  "continueIfJourneyIsNotLocked" should {
 
-      val result = new TestService(false).continueIfJourneyIsNotLocked(testJourneyId, testInternalId)(continueResult)
+    "redirect to the KnownFactsMismatchWithin24hrs Error Page" when {
+      "isJourneyLocked method returns 'true'" in {
+        enable(KnownFactsCheckFlag)
+        mockGetVrnInfo(testJourneyId, testInternalId)(Future.successful(Some(testVatNumber)))
+        mockIsVrnBlocked(testVatNumber)(Future.successful(true))
 
-      result.map(_.header.status mustBe 200)
+        val result = TestService.continueIfJourneyIsNotLocked(testJourneyId, testInternalId)(continueResult)(request)
 
-      contentAsString(result) shouldBe "next action executed"
+        result.map(_.header.status mustBe Some(SEE_OTHER))
+        result.map(_.header.headers.get("location") mustBe "/claim-vat-enrolment/error/access-still-locked")
+      }
     }
-  }
 
-  "the journey redirect to errorPage " when {
-    "knownFactsCheck is enabled" in {
-      enable(KnownFactsCheckFlag)
+    "redirect to the successive Page" when {
+      "isJourneyLocked method returns 'false'" in {
+        enable(KnownFactsCheckFlag)
+        mockGetVrnInfo(testJourneyId, testInternalId)(Future.successful(Some(testVatNumber)))
+        mockIsVrnBlocked(testVatNumber)(Future.successful(false))
+
+        val result = TestService.continueIfJourneyIsNotLocked(testJourneyId, testInternalId)(continueResult)(request)
+
+        result.map(_.header.status mustBe Some(SEE_OTHER))
+        result.map(_.header.headers.get("location") mustBe "/claim-vat-enrolment/error/access-still-locked")
+      }
 
-      mockGetVrnInfo(testJourneyId, testInternalId)(Future.successful(None))
+      "isJourneyLocked method returns 'false', no record exists at Invalid-Submission Repo" in {
```

Comment:
This is under the `'"redirect to the successive Page" when {'` umbrella which isn't accurate